### DISCUSSION TITLE
Plasma 6 no sound module

### DISCUSSION THREAD

**Discussion by 01fiore** [mocaccinoOS/mocaccino#141] [General] [answered]
Good evening, I installed the ISO KDE plasma 6, in systemsetting the sound section is empty, it lacks the PulseAudio module "module-gsettings".

$ sudo luet search pulseaudio
 INFO   --- Search results (pulseaudio): ---
> apps/pulseaudio-qt-1.5.0+1
 -> Category: apps
 -> Name: pulseaudio-qt
 -> Version: 1.5.0+1
 -> Description: Qt bindings for libpulse
 -> Repository: mocaccino-community-stable
 -> Uri: https://invent.kde.org/libraries/pulseaudio-qt
 -> Installed: false
> apps/pulseaudio-qt-1.5.0+1
 -> Category: apps
 -> Name: pulseaudio-qt
 -> Version: 1.5.0+1
 -> Description: Qt bindings for libpulse
 -> Repository: mocaccino-community-stable
 -> Uri: https://invent.kde.org/libraries/pulseaudio-qt
 -> Installed: false

$ sudo luet search pipewire
 INFO   --- Search results (pipewire): ---
> entity/pipewire-6
 -> Category: entity
 -> Name: pipewire
 -> Version: 6
 -> Description:
 -> Repository: mocaccino-os-commons-stable
 -> Uri:
 -> Installed: true
> system-profile/pipewire-0.2
 -> Category: system-profile
 -> Name: pipewire
 -> Version: 0.2
 -> Description: Automatically setup pipewire on boot
 -> Repository: mocaccino-os-commons-stable
 -> Uri:
 -> Installed: true

Merci.

**Comment by joostruis** [comment]
You are not going to find any answers using luet search this way. I rather think it is a sound driver issue. Can you post any information about your audio chipset?

**Reply by 01fiore** [reply]
$ lspci
00:00.0 Host bridge: Intel Corporation Xeon E3-1200 v5/E3-1500 v5/6th Gen Core Processor Host Bridge/DRAM Registers (rev 07)
00:01.0 PCI bridge: Intel Corporation 6th-10th Gen Core Processor PCIe Controller (x16) (rev 07)
00:08.0 System peripheral: Intel Corporation Xeon E3-1200 v5/v6 / E3-1500 v5 / 6th/7th/8th Gen Core Processor Gaussian Mixture Model
00:14.0 USB controller: Intel Corporation 100 Series/C230 Series Chipset Family USB 3.0 xHCI Controller (rev 31)
00:14.2 Signal processing controller: Intel Corporation 100 Series/C230 Series Chipset Family Thermal Subsystem (rev 31)
00:16.0 Communication controller: Intel Corporation 100 Series/C230 Series Chipset Family MEI Controller #1 (rev 31)
00:17.0 SATA controller: Intel Corporation Q170/Q150/B150/H170/H110/Z170/CM236 Chipset SATA Controller [AHCI Mode] (rev 31)
00:1c.0 PCI bridge: Intel Corporation 100 Series/C230 Series Chipset Family PCI Express Root Port #1 (rev f1)
00:1c.2 PCI bridge: Intel Corporation 100 Series/C230 Series Chipset Family PCI Express Root Port #3 (rev f1)
00:1f.0 ISA bridge: Intel Corporation Z170 Chipset LPC/eSPI Controller (rev 31)
00:1f.2 Memory controller: Intel Corporation 100 Series/C230 Series Chipset Family Power Management Controller (rev 31)
00:1f.3 Audio device: Intel Corporation 100 Series/C230 Series Chipset Family HD Audio Controller (rev 31)
00:1f.4 SMBus: Intel Corporation 100 Series/C230 Series Chipset Family SMBus (rev 31)
01:00.0 VGA compatible controller: NVIDIA Corporation GP107 [GeForce GTX 1050 Ti] (rev a1)
01:00.1 Audio device: NVIDIA Corporation GP107GL High Definition Audio Controller (rev a1)
02:00.0 USB controller: ASMedia Technology Inc. ASM1142 USB 3.1 Host Controller
03:00.0 Ethernet controller: Qualcomm Atheros Killer E2400 Gigabit Ethernet Controller (rev 10)

**Reply by joostruis** [reply]
Could you try running this as a user in the terminal:
`systemctl --user --now start pipewire.service wireplumber.service pipewire-pulse.service`

**Reply by 01fiore** [reply]
After putting the command, everything works perfectly, merci.

**Comment by joostruis** [comment]
Ok tracking the issue here now. https://github.com/mocaccinoOS/mocaccino/issues/142

Thanks for reporting!

**Reply by 01fiore** [reply]
Thanks to you have a good evening.

**Comment by joostruis** [comment, marked as answer]
It seems that a service that should handle profiles isn't installed. To fix this on your system:

`luet install utils/yip-systemd`
and then do
`systemctl enable yip-boot`

Now reboot and it should be resolved.

**Reply by 01fiore** [reply]
We tried in two simultaneously plasma 6, we had the same problem, everything works perfectly with the orders you posted.

**Reply by joostruis** [reply]
I am going to release version 1.7.1 with this nasty issue resolved. Thanks again for reporting!
